Sources/iPad/View Controllers/File/DocumentsViewController_iPad.m

```diff
@@ -424,4 +424,19 @@ - (void) updateLabelsWithNewLanguage{
     _navigation.topItem.title = Localize(@"Documents") ;
 }
 
+#pragma mark - handle orientation
+
+-(void) didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceOrientation {
+    [super didRotateFromInterfaceOrientation:fromInterfaceOrientation];
+    float height;
+    if (UIInterfaceOrientationIsPortrait([[UIDevice currentDevice] orientation])){
+        height = MAX([UIScreen mainScreen].bounds.size.width,[UIScreen mainScreen].bounds.size.height);
+    } else {
+        height = MIN([UIScreen mainScreen].bounds.size.width,[UIScreen mainScreen].bounds.size.height);
+    }
+    [self.view setFrame:CGRectMake(self.view.frame.origin.x, 0, self.view.frame.size.width, height)];
+    
+}
+
```

Comment:
This code is copied in 3 different classes. Find a way to reuse the same method in all classes. For ex you could have a common method that returns a `CGRectFrame` to use in `setFrame`.